ci/env

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-export RUSTC_VERSION=nightly-2022-12-18
+export RUSTC_VERSION=nightly-2023-04-16
```

Comment:
Q: is this required b/c of the changes or simply bumping the compiler version for good measure?